topics/genome-annotation/tutorials/introduction/slides.html

```diff
@@ -161,7 +161,7 @@
 - Integration of evidence and *ab initio* predictions
 - "Consensus" of multiple sources
 - Automated pipelines
-  - [**Maker**]({% link topics/genome-annotation/tutorials/annotation-with-maker-short/tutorial.md %}), **Braker**, [**Funannotate**]({% link topics/genome-annotation/tutorials/funannotate/tutorial.md %}), **Pasa**, **Prokka**, ...
+  - [**Maker**]({% link topics/genome-annotation/tutorials/annotation-with-maker-short/tutorial.md %}), **Braker**, **Braker3**, [**Funannotate**]({% link topics/genome-annotation/tutorials/funannotate/tutorial.md %}), **Helixer** **Pasa**, **Prokka**, ...
```

Comment:
I wouldn't write Helixer here as it doesn't do reconciliation

Reply:
By the way, there's a typo on the title of this slide : there's only 1 `l` in reconciliation